src/extensions/mountainview/MVSortingView/ViewContainer.tsx

```diff
@@ -18,6 +19,7 @@ const ViewContainer: FunctionComponent<Props> = ({ children, views, onViewClosed
     const [currentNorthView, setCurrentNorthView] = useState<View | null>(null)
     const [currentSouthView, setCurrentSouthView] = useState<View | null>(null)
     const [activeArea, setActiveArea] = useState<'north' | 'south'>('north')
+    const [splitterFrac, setSplitterFrac] = useState(0.5)
```

Comment:
Keep track of the splitter frac (Rather than the absolute possition) in order to work well with window resizes.